src/restful/controllers/authControllers.js

```diff
@@ -22,6 +22,7 @@ class AuthController {
   static async loginCallback(req, res) {
     try {
       req.user.role = "developer";
+      req.user.isLoggedIn = true;
```

Comment:
@Nkbtemmy `isLoggedIn` will be updated only once user doesn't exist because of the `findOrCreateUser` service doesn't update the user.